public_dropin_environments/java_codegen/dr_requirements.txt

```diff
@@ -1,2 +1,2 @@
 pyarrow==0.14.1
-datarobot-drum==1.5.6
+datarobot-drum==1.5.6.post1
```

Comment:
why `1.5.6.post1` and not `1.5.7`?

Reply:
Small change since the last release, which I only want to test on the Java env.
No need to update all the envs first on staging then on prod.